app/src/main/java/emu/skyline/EmulationActivity.kt

```diff
@@ -574,6 +582,50 @@ class EmulationActivity : AppCompatActivity(), SurfaceHolder.Callback, View.OnTo
         vibrators[index]?.cancel()
     }
 
+    @Suppress("unused")
+    fun showKeyboard(buff : ByteBuffer, initialText : String) : SoftwareKeyboardDialog? {
```

Comment:
The **Buff**-er strikes again, I do think that naming it `buffer` is just neater overall.